src/app/core/admin/admin-menu/admin-menu-item-details/admin-menu-item-details.component.html

```diff
@@ -1,34 +1,48 @@
 <h2 mat-dialog-title>
   @if (isNew) {
-    <ng-container i18n>Add New Menu Item</ng-container>
+    @if (isShortcut) {
+      <ng-container i18n>Add New Shortcut</ng-container>
+    } @else {
+      <ng-container i18n>Add New Menu Item</ng-container>
+    }
```

Comment:
```suggestion
    <ng-container i18n="menu item edit title">Add New Item</ng-container>
```

Let's simplify template and translation complexity here